packages/node_modules/@ciscospark/react-component-utils/src/metrics.js

```diff
@@ -36,6 +36,7 @@ export function sendPerfMetric({key, data}, spark) {
       hostname: window.location.hostname,
       pathname: window.location.pathname,
       widgetVersion: process.env.REACT_CISCOSPARK_VERSION,
+      jsSdkVersion: spark.version,
```

Comment:
this should be done automatically by the sdk, probably [here](https://github.com/ciscospark/spark-js-sdk/blob/master/packages/plugin-metrics/src/client-metrics-batcher.js#L12-L15)